servicex_app/tests/resources/transformation/test_submit.py

```diff
@@ -210,7 +212,8 @@ def test_submit_transformation_default_scheme(self, mock_rabbit_adaptor,
                                                   mock_dataset_manager_from_did,
                                                   mock_codegen,
                                                   mock_app_version):
-        client = self._test_client(rabbit_adaptor=mock_rabbit_adaptor, code_gen_service=mock_codegen)
```

Comment:
In future, please avoid reformatting unrelated code - it makes the PR so much more noisy and hard to review